lib/structures/GroupChannel.js

```diff
@@ -12,7 +12,6 @@ const User = require("./User");
 * @prop {String} id The ID of the channel
 * @prop {String} mention A string that mentions the channel
 * @prop {Call?} call The current group call, if any
-* @prop {Call?} lastCall The previous group call, if any
```

Comment:
My two cents: Bots can't join group channels, and very few bots are still in them (only bot accounts converted from user accounts way back when, which were already in group channels prior to being converted). I think it makes more sense to get rid of this class entirely, and in any case where a GroupChannel would be necessary, just no-op instead (ignore relevant events, refuse to add these channels to cache, etc.)